src/components/Icons/Api.tsx

```diff
@@ -0,0 +1,10 @@
+const Api: React.FC = () => (
```

Comment:
Just by looking at the name I don't understand what it is: a component or an icon. Maybe would be good to rename this component